.github/workflows/test.yml

```diff
@@ -22,9 +22,18 @@ jobs:
         with:
           toolchain: stable
 
+      - name: Install cargo-binstall
```

Comment:
let's not worry about having this running from the CI, we can create a follow up issue/pr for this :)

Reply:
done